test/runtests.jl

```diff
@@ -2,13 +2,10 @@
 using LightGraphs
 using Base.Test
 
-
 g1 = smallgraph(:petersen)
 g2 = smallgraph(:tutte)
 g3 = PathGraph(5)
 g4 = PathDiGraph(5)
-g5 = DiGraph(4)
-add_edge!(g5,1,2); add_edge!(g5,2,3); add_edge!(g5,1,3); add_edge!(g5,3,4)
 g6 = smallgraph(:house)
```

Comment:
this is bad practice, object definition should stay near where they are used. I run into errors when defined a g5 graph to add a test in subgraph, and this lead to failures in successively included files. It took a while to figure out the cause of that